src/detext/utils/misc_utils.py

```diff
@@ -406,84 +406,52 @@ def estimate_train_steps(input_pattern, num_epochs, batch_size, isTfrecord):
-
-    # # for job search latency test
-    # input_file = '/home/wguo/projects/bert/job/dev.tfrecords'
-    # output_file = '/home/wguo/projects/bert/job/dev.fake'
-    # target_docs = 1000
-    # num_wide = 96
-    # field_names = ['wide_ftrs', 'doc_job_company', 'doc_job_title']
-    # generate_latency_test_data(input_file, output_file, field_names, target_docs, num_wide)
-
-    # random baseline
-    # input_files = [
-    #     '/home/wguo/data/people_v6/NAV/test/00000',
-    #     '/home/wguo/data/people_v6/NAV/test/00001',
-    #     '/home/wguo/data/people_v6/NAV/test/00002',
-    #     '/home/wguo/data/people_v6/NAV/test/00003',
-    #     '/home/wguo/data/people_v6/NAV/test/00004',
-    # ]
-    # random_baseline(input_files, 10)
-
-    # input_files = [
-    #     '/home/wguo/projects/bert/people_v4/00006',
-    #     # '/home/wguo/projects/bert/people_v4/00001',
-    # ]
-    # data_stats(input_files)
-
-    # for job search latency test
-    # input_file = '/tmp/jsdata/dev-00000.tfrecords'
-    # output_file = '/tmp/jsdata/dev-00000.tfrecords.1000'
-    # target_docs = 1000
-    # num_wide = 95
-    # field_names = ['wide_ftrs', 'doc_job_company', 'doc_job_title']
-    # generate_latency_test_data(input_file, output_file, field_names, target_docs, num_wide)
-
-    # for people search latency test
-    # input_file = '/home/xwli/ps-data/output-00000.tfrecords'
-    # output_file = '/home/xwli/ps-data/output-00000.tfrecords.200'
-    # target_docs = 200
-    # num_wide = 167
-    # field_names = ['wide_ftrs', 'doc_headlines', 'doc_currCompanies', 'doc_currTitles', 'doc_prevCompanies', 'doc_prevTitles']
-    # generate_latency_test_data(input_file, output_file, field_names, target_docs, num_wide)
-
-    # sample 1 tfrecord
-    # input_file = '/tmp/psdata/L2/output-00003.tfrecords'
-    # output_file = '/tmp/psdata/L2/sample1.tfrecords'
-    # sample_tfrecords(input_file, 1, output_file)
-    input_file = '/tmp/psdata/L2/test/output-00000.tfrecords'
-    output_file = '/tmp/psdata/L2/query_embedding_samples.32d.test'
-    num_samples = 20
-    model_base_dir = '/home/xwli/models/ps-models/l2/expts_v2/ps-LiBERT-3-layer-32d-5df-100h-TrueDp-0.3TrainData-1'
-    query_embedding_model = model_base_dir + '/savedmodel_query_embedding_only'
-    generate_unit_test_query_embeddings(input_file, query_embedding_model, output_file, num_samples)
+def _bytes_feature(value):
+    """Returns a bytes_list feature"""
+    if isinstance(value, type(tf.constant(0))):
+        value = value.numpy()  # BytesList won't unpack a string from an EagerTensor.
+    return tf.train.Feature(bytes_list=tf.train.BytesList(value=value))
+
+
+def _float_feature(value):
+    """Returns a float_list feature"""
+    return tf.train.Feature(float_list=tf.train.FloatList(value=value))
+
+
+def _int64_feature(value):
+    """Returns an int64_list feature"""
+    return tf.train.Feature(int64_list=tf.train.Int64List(value=value))
+
+
+def create_sample_tfrecord(out_file):
```

Comment:
If these functions are utils for testing, shall we move it to `test_utils.py`? It'll be good to clean up some functions in `test_utils.py` if needed.

Reply:
Moved to test_utils.py in latest commit